recipe-server/client/control_new/tests/components/forms/test_FormItem.js

```diff
@@ -16,8 +22,21 @@ describe('<FormItem>', () => {
   };
 
   it('should work', () => {
-    const wrapper = () => shallow(<FormItem {...props} />);
+    const wrapper = () => shallow(<WrappedFormItem {...props} />);
 
     expect(wrapper).not.toThrow();
   });
+
+  it('should correctly trim whitespace', () => {
+    const formItemInstance = new FormItem();
+
+    const whitespaceBefore = createFakeEvent('   foobar');
+    const whitespaceAfter = createFakeEvent('foobar   ');
+    const noWhitespace = createFakeEvent('foobar');
```

Comment:
Super nit: could use another test here against `foo bar` or similar, to ensure spaces aren't removed within strings, only ends.